config/constants.ts

```diff
@@ -200,8 +200,8 @@ export const mockInstrumentRunTableName = 'stacky-instrument-run-table';
 export const mockInputMakerTableName = 'stacky-input-maker-table';
 export const mockWorkflowManagerTableName = 'stacky-workflow-manager-table';
 export const mockIcav2ProjectIdSsmParameterName = '/orcabus/stacky/icav2_project_id_and_name_json';
-export const mockPrimaryOutputUriSsmParameterName = '/orcabus/stacky/primary_output_uri'; // icav2://7595e8f2-32d3-4c76-a324-c6a85dae87b5/primary_data/__instrument_run_id__/__portal_run_id__/
-export const mockAnalysisOutputUriSsmParameterName = '/orcabus/stacky/analysis_output_uri'; // icav2://7595e8f2-32d3-4c76-a324-c6a85dae87b5/analysis_data/__workflow_name__/__workflow_version__/__portal_run_id__/
-export const mockAnalysisLogsUriSsmParameterName = '/orcabus/stacky/analysis_logs_uri'; // icav2://7595e8f2-32d3-4c76-a324-c6a85dae87b5/analysis_logs/__workflow_name__/__workflow_version__/__portal_run_id__/
+export const mockPrimaryOutputUriSsmParameterName = '/orcabus/stacky/primary_output_uri'; // icav2://7595e8f2-32d3-4c76-a324-c6a85dae87b5/primary/__instrument_run_id__/__portal_run_id__/
+export const mockAnalysisOutputUriSsmParameterName = '/orcabus/stacky/analysis_output_uri'; // icav2://7595e8f2-32d3-4c76-a324-c6a85dae87b5/analysis/__workflow_name__/__portal_run_id__/
```

Comment:
Just confirming; do we want to remove `-data` suffix from the mid path too?

No strong preference from me. If we decided so, I will update [the design doc](https://umccr.slack.com/archives/C025TLC7D/p1718269513888569?thread_ts=1717638564.248829&cid=C025TLC7D) accordingly.

@pdiakumis @reisingerf 

This must be the last bit that I might have missed yesterday discussion, Sorry. Let us finalise here, folks.

Reply:
No strong feelings either, but I'd personally would keep the "-data". 
It's more specifically talking about "data" and is less easily confused. For example AWS has primary (and secondary) analysis workflows, the outputs of which would NOT fall under our "primary" definition.

Reply:
Not fussed either. Compare:

- `analysis-data`
- `analysis-cache`
- `primary-data`

vs.

- `analysis`
- `analysis-cache`
- `primary`

Reply:
Okay. @alexiswl any preference about `-data` suffix?

Reply:
I've gathered the feedback. Team members' view are flexible on either way. So, making a call here as tiebreaker.

Let us go without `-data` suffix for this round of iteration for the mid-path. So, just `/primary/` and `/analysis/`. I will update the orcabus design doc.

In this case, no change is needed here anymore.